Podfile

```diff
@@ -43,7 +43,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-    pod 'WordPressKit', '~> 4.6.0'
+    pod 'WordPressKit', '~> 4.7.0-beta.1'
```

Comment:
Authenticator now has a dependency to WordPressKit version 4.7.0-beta.1. Since the dependency in Authenticator and WPiOS cannot be out of sync, the updated pod version is here as well.